brave/brave-ios
# [v1.60] Fix #8182: KeyringInfo Refactor, Bump BraveCore to v1.60.81

## Summary of Changes
- `accountsInfo` was removed from `KeyringInfo`. We must fetch accounts using `GetAllAccounts` api on Keyring Service now.

This pull request fixes #8182

## Submitter Checklist:


## Test Plan:
1. Test setting up new wallet & wallet restore 
2. Verify Ethereum & Solana accounts shown on Accounts tab
3. Test creating a send transaction
4. Verify Transaction Confirmation displays as expected (accounts shown in header)
5. Verify transaction shows on Activity tab
6. Verify accounts for token coin type shown in Asset Detail, transaction created in step 3 shows up
7. Tap the transaction and verify transaction detail displays as expected


## Screenshots:



## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `bug` / `enhancement`
